data_preprocessing/collectivite_territoriale.py

```diff
@@ -1,15 +1,10 @@
-import os
 import zipfile
 
 import pandas as pd
 import requests
 
 
-def preprocess_colter_data(
-    data_dir,
-    **kwargs,
-) -> None:
-    os.makedirs(os.path.dirname(data_dir), exist_ok=True)
```

Comment:
why do we remove that ? If data_dir does not exist in container, dag will fail

Reply:
I don't save the data anymore. Everything is done on SQLite.
I created a `historique` branch with the older code and DAGs.
But in this code, we will not be using any update DAGs as they do not pass for "établissements".